### PR TITLE
chore: add renamed scripts/release.sh in d9bb604 and troubleshooting

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

It seems a renamed release script is missing.
https://github.com/appium/appium/commit/d9bb604fb62282b57ebc55d650289d103299b20a

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by KazuCocoa** [comment]
Will update documentation in this PR

File: docs/en/contributing-to-appium/release-appium.md [hunk 1]
```diff
@@ -20,4 +20,21 @@ Appium follows the GitLab flow approach. Releases are made on release branches t
...
+# Troubleshooting
+
+## When you publish a version in wrong channel
+
+For example, you accidentally pusbed `rc` version in `latest` channel.
+Then, you can make the version `deprecate`, and re-tag the previous stable version in order to keep the stable one via `npm install`.
+`dist-tag` helps it.
+
+```
+$ npm deprecate appium@1.21.0-rc.0 "this release should be in rc channel" --tag=lates
```
Comment (jlipps) [inline docs/en/contributing-to-appium/release-appium.md:34]: `--tag=latest`?

**Review by jlipps — APPROVED** [review]

File: docs/en/contributing-to-appium/release-appium.md [hunk 2]
```diff
@@ -20,4 +20,21 @@ Appium follows the GitLab flow approach. Releases are made on release branches t
 1. Create a new release on GitHub: go to `https://github.com/appium/appium/releases/tag/v<VERSION>` and hit "Edit Tag". Make the release name `<VERSION>` (e.g., `2.0.5`), then paste in the changelog (but not the changelog header for this version). If it's a beta release, mark as pre-release.
 1. Create a new post on discuss.appium.io announcing the release. Post it in the "News" category. Paste in the changelog and any choice comments. Pin it and unpin the previous release post.
 1. Begin process of releasing `appium-desktop`.
-1. Notify @jlipps to so he can tweet a link to the discuss post.
\ No newline at end of file
+1. Notify @jlipps to so he can tweet a link to the discuss post.
+
+# Troubleshooting
+
+## When you publish a version in wrong channel
```
Comment (mykola-mokhnach) [inline docs/en/contributing-to-appium/release-appium.md:27]: to a wrong channel

File: docs/en/contributing-to-appium/release-appium.md [hunk 3]
```diff
@@ -20,4 +20,21 @@ Appium follows the GitLab flow approach. Releases are made on release branches t
...
+
+For example, you accidentally pusbed `rc` version in `latest` channel.
```
Comment (mykola-mokhnach) [inline docs/en/contributing-to-appium/release-appium.md:29]: into the `latest`

File: docs/en/contributing-to-appium/release-appium.md [hunk 4]
```diff
@@ -20,4 +20,21 @@ Appium follows the GitLab flow approach. Releases are made on release branches t
...
+Then, you can make the version `deprecate`, and re-tag the previous stable version in order to keep the stable one via `npm install`.
```
Comment (mykola-mokhnach) [inline docs/en/contributing-to-appium/release-appium.md:30]: you could add the `deprecated` label to this version

File: docs/en/contributing-to-appium/release-appium.md [hunk 5]
```diff
@@ -20,4 +20,21 @@ Appium follows the GitLab flow approach. Releases are made on release branches t
...
+`dist-tag` helps it.
```
Comment (mykola-mokhnach) [inline docs/en/contributing-to-appium/release-appium.md:31]: ... command helps to achieve that

File: docs/en/contributing-to-appium/release-appium.md [hunk 6]
```diff
@@ -20,4 +20,21 @@ Appium follows the GitLab flow approach. Releases are made on release branches t
...
+
+```
+$ npm deprecate appium@1.21.0-rc.0 "this release should be in rc channel" --tag=latest
+$ npm dist-tag add appium@1.20.2 latest
+```
+
+You can confirm if the above step works with calling `npm install -g appium`.
```
Comment (mykola-mokhnach) [inline docs/en/contributing-to-appium/release-appium.md:38]: by calling

File: scripts/release.sh [hunk 7]
```diff
@@ -0,0 +1,5 @@
+npm version prerelease --preid=$1
+git push
+git push --tags
+# Note: Do not forget to add `--tag rc` for RC channel. Without the tag, the publishment will be in `latest`.
```
Comment (mykola-mokhnach) [inline scripts/release.sh:4]: will end up in

**Comment by KazuCocoa** [comment]
btw, thanks @dpgraham for the release step. Your documentation and script helped me very much!

**Pull request merged**